contrib/macos/create-app-bundle.sh

```diff
@@ -35,6 +35,16 @@ install -m 644 "README"                                    "$dst/SharedSupport/m
 install -m 644 "docs/README.video"                         "$dst/SharedSupport/video.txt"
 # install -d     "$dst/Contents/_CodeSignature/" # ?
 
+install -m 755 "contrib/macos/Start DOSBox.command"        "dist/"
```

Comment:
Maybe this file should be named `Start DOSBox Staging.command`? 

Upstream and DOSBox-X don't (yet) support the `--working-dir` argument, so by using unspecified DOSBox term here (and not our project name), we might have people complain: _"DOSBox Staging is bogarting the startup `.command` for *ALL* the DOSBoxes!"_

Reply:
Sure, I can rename it. I was under the impression we want to move towards calling it just DOSBox, but you're right, that hasn't happened yet and it's best to the clarify which DOSBox variant we're launching with the command.